spec/core/transaction/schemas/TransactionStatusEnum.yml

```diff
@@ -181,11 +183,13 @@ description: |
   * Failure_Core_Address_Collision - Validation failed because an address collision is detected.
   * Failure_Core_Importance_Block_Mismatch - Validation failed because the block does not match the schema of an importance block.
   * Failure_Core_Unexpected_Block_Type - Validation failed because the block type is unexpected.
+  * Failure_Core_Block_Explicit_Transactions_Hash_Mismatch - Validation failed because a block did not have the expected transactions hash at a specified height.
```

Comment:
did -> does?
transactions hash ->  explicit transactions hash?

Reply:
I copied the description from the c++ comment like the other codes

https://github.com/symbol/catapult-client/blob/dev/plugins/coresystem/src/validators/Results.h#L80

Should we update both places?